### PR TITLE
[NOJIRA] Get description lists over the line

### PR DESCRIPTION
**Pull request by matthewdavidson**
**Note:** I have renamed `Definition` -> `Description` as per Mozilla's terminology as this is the source of truth we use (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl).

I also added LTR support and modified the indent margin for `<dd>`'s.

Here is what it looks like:

image

### CODE REVIEW

**Comment by matthewdavidson** [comment]
cc @janeklb - i hope you don't mind. I wanna get this component released :)

**Comment by janeklb** [comment]
not at all, got plenty of things to do :)

File: .storybook/config.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ configure(() => {
   require('./../packages/bpk-component-chip/stories');
   require('./../packages/bpk-component-close-button/stories');
   require('./../packages/bpk-component-code/stories');
-  require('./../packages/bpk-component-definition-list/stories');
+  require('./../packages/bpk-component-description-list/stories');
```
Comment (shaundon) [inline .storybook/config.js]: Just needs to be one line lower, as right now it's appearing before `bpk-component-content-container`.

Comment (matthewdavidson) [reply .storybook/config.js]: Good catch - should actually be 2 lines lower 😉  😂

Comment (shaundon) [reply .storybook/config.js]: Close enough :P

**Pull request merged**